backend/isoko-backend/__tests__/unit/handlers/business/put-edit-business-page.test.js

```diff
@@ -0,0 +1,509 @@
+                city: 'Venice Beach', 
+                state: 'CA',
+                street: '70 Newport Pier', 
+                zip: '92663',
+                type: 'B&M',
+                tags: ['Women-Owned'],
+                keywords:['Desserts'],
+                shortDesc: '', 
+                businessId: '-664125567',
+                hours: {}, 
+                links: {}, 
+                aboutOwner: {
+                    owner: '',
+                    ownerName: 'Lindsay Bluth', 
+                    ownerPhone: '123-456-7890',
+                    ownerDesc: '',
+                    ownerPhoto: ''
+                },
+                reviews: [], 
+                photos: [],
+                lister: 'George Michael Bluth'
+            }; 
+
+            // act
+            const result = await putEditBusinessPageHandler(event);
+
+            // assert
+            expect(result.body.results).toEqual(expectedItems);
+            expect(updateSpy).toHaveBeenCalledWith({
+               TableName: BUSINESS_TABLE,
+               Key: {
+                "businessId": '-664125567'
+                }, 
+               UpdateExpression: `set aboutOwner.ownerName = :a`,
+               ExpressionAttributeValues: { 
+                ":a": 'Lindsay Bluth'
+               },
+               ReturnValues: "UPDATED_NEW"
+            });
+        }); 
+
+        it('Should update value of multiple target fields', async () => {
+            // arrange
+            mockUpdateResults = 
+            {
+                pk: '-664125567',
+                sk: 'INFO',
+                name: "Lindsay Bluth's Original Frozen Banana",
+                city: 'Venice Beach', 
+                state: 'CA',
+                street: '70 Newport Pier', 
+                zip: '92663',
+                type: 'B&M',
+                tags: ['Women-Owned'],
+                keywords:['Desserts'],
+                shortDesc: '', 
+                businessId: '-664125567',
+                hours: {}, 
+                links: {}, 
+                aboutOwner: {
+                    owner: '', 
+                    ownerName: 'Lindsay Bluth', 
+                    ownerPhone: '123-456-7890', 
+                    ownerDesc: '', 
+                    ownerPhoto: ''
+                },
+                reviews: [], 
+                photos: [],
+                lister: 'George Michael Bluth'
+            }; 
+
+            updateSpy.mockReturnValue({
+                promise: () => Promise.resolve({ Items: mockUpdateResults }),
+             });
+
+            const event = {
+                httpMethod: 'PUT',
+                pathParameters: {
+                    'businessId': '-664125567'
+                }, 
+                body: JSON.stringify({
+                    'name' : "Lindsay Bluth's Original Frozen Banana", 
+                    'aboutOwner' : {
+                        'ownerName': 'Lindsay Bluth'
+                    }
+                })
```

Comment:
```
body: JSON.stringify({
                    'name' : "Lindsay Bluth's Original Frozen Banana", 
                    'aboutOwner' : {
                        'ownerName': 'Lindsay Bluth',
                        'somethingElse': 'Another Value'
                    },
                    'another': 'val'
                })
```

This test case will give the result of

`"UpdateExpression": "set name = :a, aboutOwner.ownerName = :b, aboutOwner.somethingElse = :c, another = :b"`